scripts/circle-test-postgres.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+function exit_if_fail {
+    command=$@
+    echo "Executing '$command'"
+    eval $command
+    rc=$?
+    if [ $rc -ne 0 ]; then
+        echo "'$command' returned $rc."
+        exit $rc
+    fi
+}
+
+export GRAFANA_TEST_DB=postgres
+
+time for d in $(go list ./pkg/...); do
```

Comment:
What is the benefit of the for loop versus just doing `go test ./pkg/...`? Is there some db reset code that is run in between each package?